### PR TITLE
fix ebpf verifier error and offsets parsing

### PR DESCRIPTION
**Pull request by MarioHewardt**

### CODE REVIEW

File: generateUnameOffsets.py [hunk 1]
```diff
@@ -109,12 +109,12 @@
         output = '    '
         if not first:
             output += 'else '
-        output += f'if (strcmp(param, "{param}"))\n'
+        output += f'if (strcmp(param, "{param}")==0)\n'
```
Comment (johnstep) [inline generateUnameOffsets.py:112]: Nit: Spaces around `==`?

File: generateUnameOffsets.py [hunk 2]
```diff
@@ -109,12 +109,12 @@
...
         output += f'        return o->{param};'
         print(output)
         first = False
     for param in params_opt:
-        print(f'    else if (strcmp(param, "{param}"))')
+        print(f'    else if (strcmp(param, "{param}")==0)')
```
Comment (johnstep) [inline generateUnameOffsets.py:117]: Nit: Spaces around `==`?

File: generateUnameOffsets.py (hunk 1 above)
Comment (MarioHewardt) [reply generateUnameOffsets.py]: Fixed

File: generateUnameOffsets.py (hunk 2 above)
Comment (MarioHewardt) [reply generateUnameOffsets.py]: Fixed

**Review by johnstep — APPROVED** [review]

**Pull request merged**